CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
-# 0.80.0 / 2025-02-03
+# 0.80.1 / 2025-02-19
+
+* [BUGFIX] Fix msteamsv2_configs to work with either `webhook_url` or `webhook_url_file`. #7352
+
+# 0.80.0 / 2025-02-04
```

Comment:
Correcting the release date as it was done on 4th https://github.com/prometheus-operator/prometheus-operator/releases/tag/v0.80.0